templates/webauthn.twig

```diff
@@ -24,21 +22,20 @@
             </ul>
         </div>
     {% endif %}
-    {% if regForm|length > 0 %}
+    {% if regURL|length > 0 %}
         <form id='regform' method='POST' action='{{ regURL|raw }}'>
             <input type='hidden' id='resp' name='response_id' value='0'/>
             <input type='hidden' id='data' name='attestation_client_data_json' value='nix'/>
             <input type='hidden' id='attobj' name='attestation_object' value='mehrnix'/>
             <input type='hidden' id='type' name='type' value='something'/>
             <input type='hidden' id='operation' name='operation' value='REG'/>
-            <button type='button' onClick="{{ regForm|raw }}" onsubmit='false' >{{ '{webauthn:webauthn:newTokenButton}'|trans }}</button>
+            <button type='button' id='regformSubmit'>{{ '{webauthn:webauthn:newTokenButton}'|trans }}</button>
```

Comment:
With the new user interface / templates, we're aiming at actual strings in the templates, so that it's easier to see them in context for the translators, and also we get the original text in english if there are no translations, rather than a weird tag that doesn't mean anything.